examples/vector-db/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@examples/vector-db",
```

Comment:
I feel like vector-db isn't the right name for this since there isn't actually a vector db involved. Maybe we call it "rag" or "retrieval-augemented-generation"